JWBestPracticeApps/Google IMA companion ads/Google IMA Companion Ads/AppDelegate.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AppDelegate.swift
+//  Google IMA Companion Ads
+//
+//  Created by David Almaguer on 01/02/23.
+//
+
+import UIKit
+import JWPlayerKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        
+        // Add your JW Player license key.
+        JWPlayerKitLicense.setLicenseKey("")
```

Comment:
Please change this to match the other BPA AppDelegates
`// JWPlayerKitLicense.setLicenseKey(<#key: String#>)`